instance/tests/models/test_instance.py

```diff
@@ -22,13 +22,13 @@
 
 # Imports #####################################################################
 
+from unittest.mock import call, patch, Mock
 from urllib.parse import urlparse
 
 import re
 import yaml
```

Comment:
@itsjeyd Good catch - fixed in https://github.com/open-craft/opencraft/pull/6/commits/1216ddaec678259f01de679b561031f246851f24